oneview_redfish_toolkit/api/errors.py

```diff
@@ -24,19 +24,8 @@
                        'Session.INVALID']
 
 
-class OneViewRedfishError(Exception):
+class OneViewRedfishException(Exception):
 
-    def __init__(self, msg):
+    def __init__(self, msg, status_code_error):
```

Comment:
As I had mentioned before, I think the api should not know about the HTTP status. This is a information of the "HTTP layer" (blueprint).